st2common/st2common/config.py

```diff
@@ -262,26 +262,37 @@ def register_opts(ignore_errors=False):
         cfg.StrOpt('keystone_auth_url', default=None, help='Auth endpoint for Keystone.'),
         cfg.StrOpt('cacert', default=None, help='Optional certificate to validate endpoint.'),
         cfg.BoolOpt('insecure', default=False, help='Allow insecure communication with Mistral.'),
-        cfg.FloatOpt('jitter_interval', default=1,
-                   help='Jitter interval to smooth out HTTP requests ' +
-                        'to mistral tasks and executions API.'),
-
-        cfg.StrOpt('api_url', default=None, help=('URL Mistral uses to talk back to the API.'
-            'If not provided it defaults to public API URL. Note: This needs to be a base '
-            'URL without API version (e.g. http://127.0.0.1:9101)'))
+        cfg.FloatOpt(
+            'jitter_interval', default=0,
```

Comment:
If needed, one option would also be to use float values (e.g. `0.1`) instead of totally disabling it.

Reply:
Ok. I'll test 0.1 first and then default if all is well